corehq/apps/userreports/README.md

```diff
@@ -1430,6 +1430,9 @@ Column IDs in percentage fields *must be unique for the whole report*. If you us
 
 To sum a column and include the result in a totals row at the bottom of the report, set the `calculate_total` value in the column configuration to `true`.
 
+Not supported for the following column types:
+- expression
```

Comment:
I'm pretty sure some of the others also don't support it but didn't investigate fully